### ISSUE TITLE
Difficulty passing nasogastric tube

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9817]
**Preferred term label:**
Difficulty passing nasogastric tube

**Synonyms**
NG tube; typically due to esophageal atresia or fistula

**Definition (free text, please give PubMed ID)**
This seems to deserve its own non-anatomical term because it is a symptom of the underlying problems that is observed clinically

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal Adding new term: 
- ID: HP:6000294
- label: Difficulty passing nasogastric tube
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:2917506
- def: Difficulty inserting a nasogastric tube (NGT), a flexible plastic tube that is placed through the nose and esophagus into the stomach and used to administer drugs or for feeding. There are certain circumstances that may make NGT insertion difficult. They are esophageal narrowing, comatose or intubated patients, and patients who have sustained severe facial or skull injuries.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9817
- synonyms:
